java/runtime-native/src/main/java/org/ray/spi/impl/DefaultLocalSchedulerClient.java

```diff
@@ -92,14 +92,31 @@ public void reconstructObject(UniqueID objectId, boolean fetchOnly) {
     _reconstruct_object(client, objectId.getBytes(), fetchOnly);
   }
 
+  @Override
+  public void reconstructObjects(byte[][] objectIds, boolean fetchOnly) {
+    StringBuilder builder = new StringBuilder();
+    for (byte[] item : objectIds) {
+      UniqueID id = new UniqueID(item);
+      builder.append(id.toString());
+      builder.append(" ");
+    }
+
+    RayLog.core.info("reconstruct objects " + builder.toString());
```

Comment:
We should avoid building the string here, because it's unnecessary when info-level logging isn't enabled.
If you change the parameter type to `List<UnqiueID>`, then you can just do `RayLog.core.info("Reconstructing objects {}", objectIds);` 
（Note: here we use `{}` as a placeholder to avoid unnecessary string concatenation. It's a legacy issue that most of the code in Ray isn't using this patten. If you don't mind, could you also fix other places related in this PR?)